.github/workflows/snomed-deploy.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy to SNOMED environment
+on:
+  push:
+    branches:
+      - snomed-master
+
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy'
+        required: true
+        type: choice
+        default: qa
+        options:
+          - qa
+          - dev
+
+env:
+  ENVIRONMENT: ${{ github.event.inputs.environment || 'dev'}}
+
+jobs:
+  deploy:
+    name: Deploy to SNOMED ${{ github.event.inputs.environment || 'dev'}} environment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.SNOMED_AWS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.SNOMED_AWS_SECRET }}
+          aws-region: ${{ secrets.SNOMED_AWS_REGION }}
+          role-to-assume: ${{ secrets.SNOMED_CICD_ROLE_ARN }}
+          role-duration-seconds: 900 # 15 mins
+          role-session-name: SNOMED
+
+      - name: Deploy to SNOMED ${{ github.event.inputs.environment || 'dev'}} Environment
+        run: |
+          INSTANCE_ID=$(aws ec2 describe-instances --filters "Name=tag:Name,Values=${{ github.event.inputs.environment || 'dev'}}" "Name=instance-state-name,Values=running" --query "Reservations[].Instances[].InstanceId" --output text)
+          if [ -z "$INSTANCE_ID" ]; then
+            echo "Instance not running"
+            exit 1;
+          else
+            SSM_COMMAND_TARGET_JSON=${{ toJson('[{"Key":"InstanceIds","Values":["${INSTANCE_ID}"]}]') }}
+            aws ssm send-command --document-name "AWS-RunShellScript" \
```

Comment:
Looks good to me. However, I noticed that the logs generated by the `aws ssm send-command `command contain a lot of unnecessary information, which can make it difficult to identify important information. Specifically, the logs contain a large amount of JSON that includes details about the command, its parameters, and its status.

Although nothing seems to be critical , I suggest modifying the code to output only the necessary information, such as the status of the command and the id

Reply:
@umair-fayaz updated the command to include only command id and status.